.github/workflows/pull-request.yml

```diff
@@ -21,6 +21,9 @@ env:
   PW_DIAL_NUMBER: ${{ secrets.PW_DIAL_NUMBER }}
   PW_AGENT2_USERNAME: ${{ secrets.PW_AGENT2_USERNAME }}  
   PW_AGENT2_EXTENSION_NUMBER: ${{ secrets.PW_AGENT2_EXTENSION_NUMBER }}
+  PW_SENDER_EMAIL : ${{ secrets.PW_SENDER_EMAIL }}
+  PW_EMAIL_ENDPOINT: ${{ secrets.PW_EMAIL_ENDPOINT }}
+  PW_APP_PASSWORD: ${{ secrets.PW_APP_PASSWORD }}
```

Comment:
Could we also please add it in the description, what each of these variables are used for?

Reply:
And why do we need the email end point in the secrets? Can't it be kept in the tests file itself?